client/.env.example

```diff
@@ -1,5 +1,5 @@
 NODE_ENV=development
 # This defaults to localhost in development regardless of the value set in this file
-DEFAULT_RPC=https://rpc-df.xdaichain.com/
+DF_DEFAULT_RPC=https://rpc-df.xdaichain.com/
```

Comment:
Is it worth adding mode specific env vars? https://vitejs.dev/guide/env-and-mode.html#env-files

Reply:
I really want the env files to go away, but if you think they are valuable, we could do it in a follow up PR (something like `.env.development` yeah?

Reply:
We make them go away by doing this: https://github.com/phated/afterimage/commit/97564abef6f6a62c460b348b4a0b0ca4cecf7604 ?

Reply:
Yes, but that works best if hardhat is used at the root of the project, which is very hard to do with DF currently.

Reply:
Got it, lets leave as is and if we need an `.env.development` we can add it later.